### PR TITLE
fix back button alignment in footer

### PR DESCRIPTION
**Pull request by mohitsuman**
image

### CODE REVIEW

**Comment by mohitsuman** [comment]
test this

File: browser/app.css [hunk 1]
```diff
@@ -423,7 +423,7 @@ footer {
   }
 
   .back-btn-spacer {
-    right: 183px;
+    right: 200px;
```
Comment (dgolovin) [inline browser/app.css]: this will make space between 'Back' and 'Next' 13px wider than between 'Back' and 'Download and Install'.

**Comment by dgolovin** [comment]
@mohitsuman how does it look on macOS? Is space between buttons jumping the same way with this changes?

**Comment by mohitsuman** [comment]
@dgolovin the space difference was working fine on mac os, but I encountered the spacing change when I tried on Win. I will add the class to the back button and check.

**Comment by dgolovin** [comment]
Fixed in https://github.com/redhat-developer-tooling/developer-platform-install/pull/884

**Comment by mohitsuman** [comment]
Closing this issue as the issue is fixed in https://github.com/redhat-developer-tooling/developer-platform-install/pull/884.

**Pull request closed without merging**